eslint.js

```diff
@@ -125,12 +132,20 @@ async function run() {
         return;
     }
 
+    const current = process.cwd();
```

Comment:
I don't think you can use this. Under certain situation this might not run correctly, like when calling the actions locally. We have a var passed in, like `CUSTOM_WORKING_DIRECTORY`. Also, please confirm that in `mobile`, `olc`, and any of our other micro-service repos, like this one, local checks can run with this change.

Reply:
So for the eslint-action, you'll see that we actually chdir to the workingDirectory. the jest-action doesn't do this, and so does need to change